tools/github_webhook_processor.php

```diff
@@ -195,6 +198,12 @@ function handle_pr($payload) {
 	switch ($payload["action"]) {
 		case 'opened':
 			tag_pr($payload, true);
+			if(get_pr_code_friendliness($payload) === -1){
+				$balances = pr_balances();
+				$author = $payload['pull_request']['user']['login'];
+				if(isset($balances[$author]) && $balances[$author] < 0)
+					create_comment($payload, "You currently have a higher Feature/Fix pull request ratio than the configured rate of " . $featuresPerFix . ". Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.");
```

Comment:
uh "maintainers may close this PR at will" is kinda harsh and could be abused. this system should really be 70% nagging and 30% actual consequences.